pulsar-client-cpp/python/pulsar_test.py

```diff
@@ -147,11 +159,8 @@ def test_producer_consumer(self):
         self.assertTrue(msg)
         self.assertEqual(msg.data(), b'hello')
 
-        try:
-            msg = consumer.receive(100)
-            self.assertTrue(False)  # Should not reach this point
-        except:
-            pass  # Exception is expected
+        with self.assertRaises(PulsarException):
```

Comment:
In these tests, could we assert for `pulsar.Timeout` exception?

Reply:
Yes, refactored